src/firefly/java/edu/caltech/ipac/table/TableUtil.java

```diff
@@ -311,6 +311,14 @@ static List aryToList(Object val) {
         return Arrays.asList(val);
     }
 
+    /**
+     * @param cnames a string of column name(s)
+     * @return an array of column names split from a comma separated string, ignoring commas inside double-quotes
+     */
+    public static String[] splitCols(String cnames) {
+        return cnames == null ? new String[0] : cnames.split(",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)");
```

Comment:
This is really interesting! My takeaways:
1. I didn't knew about existence of regex group constructs `(?...)` - now I learned reviewing this :)
2. I should also add docstrings if writing new functions. Is there any plugin in IntelliJ to auto-generate params and return list (I remember using one in VSCode for python)?

Reply:
In IntelliJ, you can type `/**` then [return] to generate JavaDoc or JsDoc.  It works perfectly for JavaDoc.  But, for JsDoc, you may need to fix.

Reply:
Thanks!